### PR TITLE
add broken axe, cooldown, multiple items

### PR DESCRIPTION
**Pull request by RMS-dnb**
-added in the chance of the axe to break.  Default set at 10% 
-if axe breaks it is removed from inventory and replace with broken axe. 
-added cooldown for chopping trees , can be adjusted in config. 
-addition of multiple items for rewards, and can now be configured in the config.

### CODE REVIEW

File: server/server.lua [hunk 1]
```diff
@@ -51,7 +68,24 @@ AddEventHandler('rsg-gatherer:server:removeitem', function(item, amount)
...
 
+
+-- Add this event at the end of the server script
+RegisterServerEvent('rsg-gatherer:server:breakaxe')
+AddEventHandler('rsg-gatherer:server:breakaxe', function()
+    local _source = source
+    local Player = RSGCore.Functions.GetPlayer(_source)
+
+    Player.Functions.RemoveItem('axe', 1)
+    Player.Functions.AddItem('bread', 1)
```
Comment (TehGuardian) [inline server/server.lua:79]: Change item from bread to brokenaxe

Comment (RMS-dnb) [reply server/server.lua:79]: fixed

Comment (Captainoff51) [reply server/server.lua:79]: I had already changed it, RMS had given us the framework

Comment (RMS-dnb) [reply server/server.lua:79]: Ya i forgot to take out after testing :)  thanks for the heads up

**Pull request merged**